src/mmrelay/db_utils.py

```
@@ -413,67 +718,86 @@ def store_message_map(
+        logger.debug(
+            "Retrieved message map by meshtastic_id=%s: %s", meshtastic_id, result
+        )
+        if not result:
+            return None
+        try:
+            return result[0], result[1], result[2], result[3]
+        except (IndexError, TypeError) as e:
+            logger.exception(
+                "Malformed data in message_map for meshtastic_id %s: %s",
+                meshtastic_id,
+                e,
```

Ruff (0.14.3)

Redundant exception object included in `logging.exception` call

(TRY401)